src/workerd/api/node/zlib-util.c++

```diff
@@ -450,4 +547,67 @@ void ZlibUtil::ZlibStream::reset(jsg::Lock& js) {
   }
 }
 
+kj::Array<kj::byte> syncProcessBuffer(ZlibContext& ctx, GrowableBuffer& result) {
+  KJ_DEFER(ctx.close());
+
+  do {
+    // TODO: We respect the chunk size to match node, but it's a bit silly.. should we just
+    // override it for perf?
+    result.addChunk();
```

Comment:
Probably safe to ignore chunk size for the one-shot methods

Reply:
Well... probably should still validate that it is a correct value just to be safe, but likely ok to override it for perf.